typespec-extension/src/code-model-builder.ts

```diff
@@ -534,6 +547,22 @@ export class CodeModelBuilder {
     return codeModelOperation;
   }
 
+  private isMultipleContentTypes(httpOperation: HttpOperation): boolean {
+    if (httpOperation.parameters.parameters && httpOperation.parameters.parameters.length > 0) {
```

Comment:
Do we need to check for `length > 0`? `...parameters.some()` should handle empty list too right?